### PR TITLE
add atomic-router, home page, other pages templates

### LINKED ISSUE #16: Add atomic-router

### PR DESCRIPTION
**Pull request by Hellomyguest**
add atomic-router
add home page
add other pages templates

### CODE REVIEW

**Review by softwarrior — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,24 +1,76 @@
...
-
 npm run dev
 ```
 
-Quick commit with commitizen:
+## Technology Stack:
+[![StackShare](https://img.shields.io/badge/tech-stack-0690fa.svg?style=flat)](https://stackshare.io/stackshare/awesome-stacks)
 
-```javascript
+- #### Development environment: <br>**Vite** (https://vitejs.dev/)
```
Comment (makhnatkin) [inline README.md]: plz merge main inside this branch

File: src/App.tsx [hunk 2]
```diff
@@ -1,40 +1,23 @@
...
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </div>
+    <ConfigProvider
+      theme={{
+        algorithm:
+          themeValue === "DARK" ? theme.darkAlgorithm : theme.defaultAlgorithm,
```
Comment (makhnatkin) [inline src/App.tsx]: themeValue === themes.DARK ? theme.darkAlgorithm : theme.defaultAlgorithm,

File: src/features/theme-switcher/ui/ThemeSwitcher.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+  const onChange = (checked: boolean) => {
+    checked ? themeChangedFn("DARK") : themeChangedFn("LIGHT");
+  };
+
+  return (
+    <Space className="Space">
+      День
+      <Switch
+        checked={themeValue === "DARK"}
```
Comment (makhnatkin) [inline src/features/theme-switcher/ui/ThemeSwitcher.tsx]: Why is this code in two prs?
You should branch off the branch again from main

File: package.json [hunk 4]
```diff
@@ -10,29 +10,34 @@
     "prepare": "husky install"
   },
   "dependencies": {
-    "commitizen": "^4.2.5",
     "@ant-design/icons": "^4.8.0",
     "antd": "^5.0.2",
+    "atomic-router": "^0.8.0",
+    "atomic-router-react": "^0.8.1",
+    "commitizen": "^4.2.5",
```
Comment (makhnatkin) [inline package.json]: is really need in dependecies?
I hope it needed only in devDep
yes?

File: src/pages/home/page.css [hunk 5]
```diff
@@ -0,0 +1,11 @@
+.Content {
+    padding: 3.125rem;
```
Comment (makhnatkin) [inline src/pages/home/page.css]: use 3rem

**Review by makhnatkin — APPROVED** [review]

File: src/pages/home/page.css (hunk 5 above)
Comment (Hellomyguest) [reply src/pages/home/page.css]: done

File: package.json (hunk 4 above)
Comment (Hellomyguest) [reply package.json]: done

**Pull request merged**